ethcore/src/state/mod.rs

```diff
@@ -952,8 +951,58 @@ impl<B: Backend> State<B> {
 		}))
 	}
 
+	#[cfg(feature="evmbin-only")]
+	/// Populate a PodAccount map from this state.
+	/// Warning this is not for real time use
+	pub fn to_pod_slow(&self) -> PodState {
```

Comment:
nitpick -- maybe name this something more about its functionality `to_pod_full` or others.